### ISSUE TITLE
Report shows "Unavailable Workspace" to a user outside of the workspace that was invited

### ISSUE THREAD

**Issue by dangrous** [Expensify/App#51749]
If you haven’t already, check out our [contributing guidelines](https://github.com/Expensify/ReactNativeChat/blob/main/contributingGuides/CONTRIBUTING.md) for onboarding and email contributors@expensify.com to request to join our Slack channel!
___

**Version Number:**
**Reproducible in staging?:**
**Reproducible in production?:**
**If this was caught on HybridApp, is this reproducible on New Expensify Standalone?:**
**If this was caught during regression testing, add the test name, ID and link from TestRail:**
**Email or phone of affected tester (no customers):**
**Logs:** https://stackoverflow.com/c/expensify/questions/4856
**Expensify/Expensify Issue URL:**
**Issue reported by:** @trjExpensify 
**Slack conversation:** not Slack but https://github.com/Expensify/App/issues/48396#issuecomment-2429856004

## Action Performed:
Break down in numbered steps

1. Create a workspace
2. [User A] Submit an expense to the workspace
3. [User A] Click on the expense preview in the main chat to go to combined report.
4. [User A] Send a user mention containing User B (who is not a member of the workspace
5. [User A] Click "Invite" from the whisper message
6. [User B] Open the invited expense report

## Expected Result:
Report should contain the correct report title, including the name of the workspace

## Actual Result:
Report says `Unavailable workspace` in title instead of the workspace name

Details/investigation [here](https://github.com/Expensify/App/issues/48396#issuecomment-2432741937). This is a backend issue - we need to figure out how to share the policy name without inviting the user to the policy.

## Workaround:
Can the user still use Expensify without this being fixed? Have you informed them of the workaround?

Yes

## Platforms:

Which of our officially supported platforms is this issue occurring on?
- [X] Android: Standalone
- [X] Android: HybridApp
- [X] Android: mWeb Chrome
- [X] iOS: Standalone
- [X] iOS: HybridApp
- [X] iOS: mWeb Safari
- [X] MacOS: Chrome / Safari
- [X] MacOS: Desktop

## Screenshots/Videos

<details>
  <summary>Add any screenshot/video evidence</summary>
<img width="1510" alt="378952596-93b04143-7f38-4510-8e6f-0321e6313f6c" src="https://github.com/user-attachments/assets/e18128f2-5962-42a8-924b-c97979997710">
  
</details>

[View all open jobs on GitHub](https://github.com/Expensify/App/issues?q=is%3Aopen+is%3Aissue+label%3A%22Help+Wanted%22)

**Comment by trjExpensify** [comment]
Yeah, I'd be curious to hear from the team that worked on mention invites to understand what was expected here with inviting to reports/rooms-only on workspaces? How are we displaying which workspace the room is associated with? CC: @puneetlath @thienlnam if you guys have any insight.

**Comment by alexpensify** [comment]
Not overdue, waiting for feedback here

**Comment by alexpensify** [comment]
Waiting for feedback here, @trjExpensify should we move this to Slack for a bigger discussion?

**Comment by alexpensify** [comment]
## Heads up, I will be offline until Tuesday, November 12, 2024, and will not actively watch over this GitHub during that period.

I moved this to Weekly since we need a discussion. I'll start one next week.

**Comment by alexpensify** [comment]
I'll get to the discussion next week.

**Comment by alexpensify** [comment]
I didn't get to start a discussion before going OOO. I'll work on this one next week when I'm back online.

**Comment by alexpensify** [comment]
On my list for the week

**Comment by alexpensify** [comment]
Closing - there has been no movement here.

## Heads up, I will be offline until Wednesday, December 18, 2024, and will not actively watch over this GitHub during that period.

If this GitHub requires an urgent update, please ask for help in the #expensify-open-source Slack Room. If the inquiry can wait, I'll review it when I return online.

**Comment by puneetlath** [comment]
Yeah, I don't understand why we closed this. Seems to be a legit bug. @heyjennahay also seems to be experiencing it here: https://expensify.slack.com/archives/C07NMDKEFMH/p1734467521218349?thread_ts=1733161902.588669&cid=C07NMDKEFMH

**Comment by alexpensify** [comment]
Thanks for flagging. I didn't get to the discussion before the Holiday break. I'll try to review this week, but I am working limited hours.

**Comment by alexpensify** [comment]
There is no update; I'll get to this one after I'm back online from bereavement leave.

**Comment by alexpensify** [comment]
Internal Discussion: https://expensify.slack.com/archives/C07NMDKEFMH/p1736493280846109

**Comment by alexpensify** [comment]
Moving to be a Hot Pick

**Comment by flaviadefaria** [comment]
I believe this makes more sense in #convert than #migrate.

**Comment by alexpensify** [comment]
Heads up @anmurali and @danielrvidal - thoughts on moving this one to #convert?

**Comment by alexpensify** [comment]
Bump @anmurali and @danielrvidal

**Comment by puneetlath** [comment]
@alexpensify it'll probably be faster to just ask in the #convert room in slack.

**Comment by alexpensify** [comment]
I started the discussion here: 

https://expensify.slack.com/archives/C07HPDRELLD/p1739930424667169

**Comment by alexpensify** [comment]
Weekly Update: Hot Pick in #Convert now

**Comment by alexpensify** [comment]
Weekly Update: Hot Pick

**Comment by alexpensify** [comment]
Weekly Update: Hot Pick

**Comment by alexpensify** [comment]
Weekly Update: Hot Pick!

**Comment by alexpensify** [comment]
Update: Moving to Monthly, this one is still a Hot Pick but I won't be around next week. I'll change the label when I'm back. 

--- 

🚨 Heads up! I'll be offline until Monday, April 7, 2025, and won’t be actively monitoring this GitHub during that time. 

If this GitHub requires an urgent update, please reapply the `bug` label or reach out in the `#expensify-open-source` Slack room for help. Otherwise, I'll pick things up when I'm back online. Thanks!

**Comment by alexpensify** [comment]
Hot Pick!

**Comment by alexpensify** [comment]
Still a hot pick

**Comment by heyjennahay** [comment]
A quick note to let you know that this no longer seems to be an issue for me.  The report which previously showed as "unavailable" is now showing the report title.  It changed this week

![Image](https://github.com/user-attachments/assets/3edef1a0-2a09-4032-9824-e37671f50711)

**Comment by alexpensify** [comment]
Thanks, Jenna, for the update. I will close for now since this appears to be fixed in another issue. If needed, we can reopen later.

**Comment by dangrous** [comment]
Neat! I'm curious to know how this was fixed, but regardless it does indeed seem good to go
